.pre-commit-config.yaml

```diff
@@ -29,10 +29,11 @@ repos:
       - id: trailing-whitespace
         # ignoring Wills's wild changes
         exclude: README.md
+      - id: check-json
```

Comment:
Why is this added? Are there any json files in the repository to check?

Reply:
for validation https://github.com/Lightning-AI/lightning/tree/master/tests/tests_app/cli/jsons